kotsadm/api/src/kots_app/kots_app_store.ts

```
@@ -748,6 +753,14 @@ export class KotsAppStore {
         commitUrl: row.git_commit_url || "",
         gitDeployable: row.git_deployable
       };
+      if (row.kots_installation_spec) {
+        try {
+          const installationSpec = yaml.safeLoad(row.kots_installation_spec).spec as InstallationSpec;
+          versionItem.yamlErrors = installationSpec.yamlErrors;
+        } catch (err) {
+          console.log(`Failed to unmarshal installation spec yaml for sequence ${versionItem.sequence}`, err);
```

Is this supposed to throw an error rather than just console log it?